src/notation/notationtypes.h

```diff
@@ -442,6 +442,8 @@ struct FilterElementsOptions
     int voice = -1;
     const mu::engraving::System* system = nullptr;
     Fraction durationTicks{ -1, 1 };
+    Fraction beat{ -1, 1 };
+    const Measure* measure = nullptr;
```

Comment:
I feel like I'm completely missing something but it looks like this PR only _sets_ these fields, but doesn't actually _use_ them; in other words, only implements a UI and not the functionality itself. I'd expect to see some changes in `NotationElements::elements` / `NotationElements::constructElementPattern` / `NotationElements::constructNotePattern`. Did you forget to add those, or am I saying something stupid now?

Reply:
Well, I guess it is rather me being stupid ;-)
I was under the assumption that the UI was the only missing part.

Fortunately, after your hint, an easy thing to fix

Reply:
And now I can also reproduces @worldwideweary's issue